File: scratch/check_svg.js
const fs = require('fs');
const html = fs.readFileSync('checkout.html', 'utf8');

const regex = /<svg[^>]*>([\s\S]*?)<\/svg>/g;
let match;
const visas = [];
while ((match = regex.exec(html)) !== null) {
  if (match[0].includes('#1434CB')) {
    visas.push(match[0]);
  }
}

console.log(`Total Visa SVGs found: ${visas.length}`);
visas.forEach((v, i) => {
  console.log(`Visa ${i} (length ${v.length}):`);
  console.log(v);
  console.log('='.repeat(50));
});

if (visas.length >= 2) {
  console.log('Are they identical?', visas[0] === visas[1]);
}

File: scratch/check_svg.py
import re

with open('checkout.html', 'r', encoding='utf-8') as f:
    html = f.read()

# Find all SVGs that contain the signature color fill of Visa (#1434CB)
svgs = re.findall(r'<svg[^>]*>.*?</svg>', html, re.DOTALL)
visas = [s for s in svgs if '#1434CB' in s]

print(f"Total Visa SVGs found: {len(visas)}")
for i, v in enumerate(visas):
    print(f"Visa {i}: length={len(v)}")
    print(v[:150], "...", v[-150:])
    print("-" * 40)
